src/resolvers/nodeResolver.ts

```diff
@@ -27,7 +27,7 @@ const versionCache: {
 			.get(1)
 			.then((res: { results: Array<{ name: string }>; next?: string }) => {
 				const curr = _(res.results).map('name').filter(versionTest).value();
-				const tags = prev.concat(curr);
+				const tags = prev.concat(curr as string[]);
```

Comment:
@Page- this was one of your changes which was causing the typescript build to fail, so that's why it was added to this PR (I guess it was a version mismatch or something). Anyway i didn't really put much thought into fixing it, just attacked the offending statement. You're right though, the typing should be added on definition.